_schemaV2.graphql

```diff
@@ -9652,6 +9677,36 @@ type SaleEdge {
   node: Sale
 }
 
+type SaleRegistration {
+  bidder: Bidder
+
+  # A globally unique ID.
+  id: ID!
+
+  # A type-specific ID likely used as a database ID.
+  internalID: ID!
+  isRegistered: Boolean
+  sale: Sale
+}
+
+# A connection to a list of items.
+type SaleRegistrationConnection {
+  # A list of edges.
+  edges: [SaleRegistrationEdge]
+
+  # Information to aid in pagination.
+  pageInfo: PageInfo!
```

Comment:
Sorry I was slow responding to this PR. Is there a reason we have `pageInfo` here and not `pageCursor`?

Reply:
`pageInfo` is the correct nomenclature. Here's the [relay pagination spec](https://relay.dev/graphql/connections.htm#sel-GABDJAaBHBPwxL).